project_group_setup.py

```diff
@@ -58,6 +62,8 @@ class Options:
     endpoint = ENDPOINT
     user = "co_8.william_test"
     osg_co_id = OSG_CO_ID
+    unix_id = CLUSTER_ID
```

Comment:
```suggestion
    unix_gid = CLUSTER_GID
```

Just in case this script evolves it's better to be clear about which ID

Reply:
This value is for specifying which UNIX Cluster in COmanage we want to add UNIX Cluster groups to. For example, the UNIX Cluster for the test registry has an ucid of 10
![image](https://github.com/opensciencegrid/osg-comanage-project-usermap/assets/144060728/4fd432be-f32d-4dac-b55d-b6d366f8aac6)
How's this?:
```suggestion
    ucid = UNIX_CLUSTER_ID
```

Reply:
Ah, gotcha! I like your suggestion